control/dtime.py

```diff
@@ -88,3 +90,19 @@ def sample_system(sysc, Ts, method='zoh', alpha=None):
         raise ValueError("First argument must be continuous time system")
 
     return sysc.sample(Ts, method, alpha)
+
+
+def c2d(sysc, Ts, method='zoh'):
+    """(Deprecated) Return a discrete-time system; use sample_system instead"""
+    warn("The function `c2d` will be deprecated in a future release.  Use"
+         "`sample_system` (or `matlab.c2d`) instead.",
+         PendingDeprecationWarning)
+
+    #  Call the sample_system() function to do the work
+    sysd = sample_system(sysc, Ts, method)
+
+    # TODO: is this check needed?  If sysc is  StateSpace, sysd is too?
```

Comment:
I think this check isn't needed anymore. 

sample_system calls the system's 'sample' function, and StateSpace.sample always returns a stateSpace system.